src/components/session-submission/session-submission.tsx

```diff
@@ -0,0 +1,425 @@
+                                placeholder="Lead Doer of Stuff" name={titlePath} id={titlePath} />
+                            <div className="invalid-feedback">
+                                <ErrorMessage name={titlePath} />
+                            </div>
+                        </div>
+                    </div>
+                    <div className="col-sm-12 col-md-6">
+                        <div className="form-group">
+                            <label htmlFor={companyPath}>Company</label>
+                            <Field
+                                className={`form-control ${getValidationClass(companyPath)}`}
+                                placeholder="Some Startup" name={companyPath} id={companyPath} />
+                            <div className="invalid-feedback">
+                                <ErrorMessage name={companyPath} />
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div className="form-row">
+                    <div className="col-sm-12 col-md-6">
+                        <div className="form-group">
+                            <label htmlFor={phoneNumberPath}>Phone Number</label>
+                            <Field
+                                className={`form-control ${getValidationClass(phoneNumberPath)}`}
+                                type="phone" placeholder="715-123-4567" name={phoneNumberPath} id={phoneNumberPath} />
+                            <div className="invalid-feedback">
+                                <ErrorMessage name={phoneNumberPath} />
+                            </div>
+                        </div>
+                    </div>
+                    <div className="col-sm-12 col-md-6">
+                        <div className="form-group">
+                            <label htmlFor={emailPath}>Email</label>
+                            <Field
+                                className={`form-control ${getValidationClass(emailPath)}`}
+                                type="email" placeholder="email@gmail.com" name={emailPath} id={emailPath} />
+                            <div className="invalid-feedback">
+                                <ErrorMessage name={emailPath} />
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div className="form-row">
+                    <div className="col-sm-12 ">
+                        <div className="d-flex justify-content-end">
+                            <button className="btn btn-outline-primary" type="button" onClick={() => arrayHelpers.remove(index)}>Remove Presenter</button>
+                        </div>
+                    </div>
+                </div>
+            </React.Fragment>
+        )
+
+        function getValidationClass(path: string): string | void {
```

Comment:
I noticed this would put a class of undefined on elements. This should be updated in the future to return empty string